### ISSUE TITLE
Inference problem

### ISSUE THREAD

**Issue by lingheng123** [PRBonn/4DMOS#19]
when try to use your model to infer,i encountered this error. My pytorchlightning version is 1.9.4.It's the version too high?I saw the requirement is >=1.6.4
![IMG_4675](https://user-images.githubusercontent.com/88220400/229414858-76db5c7b-3d49-4654-acaa-fcac5b8bae37.JPG)

**Comment by wangfei-tech** [comment]
I use pytorchlightning version is 1.2.0 it can works corretcly.

**Comment by benemer** [comment]
Hey, good catch!

Looks like pytorch_lightning broke backward compatibility at some point since it now expects additional keys when loading a checkpoint from a LightningModule. I personally have version `1.7.7` installed.

I just [pushed](https://github.com/PRBonn/4DMOS/commit/203da9dd5a3ac5db47ab6aba6308305d955ce8dc) the fix to master. We now create an object of the MOSNet class and only load the weights from the checkpoint. Also, I fixed an import that was changed along the way.

I just tested with version `1.9.4`. Let me know if it works for you.

![image](https://user-images.githubusercontent.com/38326482/229435955-57920e23-a242-446d-92e9-125ed8c4c1a5.png)
